docs/user/include_jinja_list.md

```diff
@@ -17,6 +17,7 @@
 | feature_compliance | netutils.config.compliance.feature_compliance |
 | find_unordered_cfg_lines | netutils.config.compliance.find_unordered_cfg_lines |
 | section_config | netutils.config.compliance.section_config |
+| palo_alto_clean_newlines | netutils.config.conversion.palo_alto_clean_newlines |
```

Comment:
same as before. `paloalto` vs `palo_alto` in the naming.  Should use `paloalto` since that is the pattern already existing.

Reply:
Addressed in latest push